src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -51,48 +53,118 @@ public boolean isMaintainer() {
     return this.isMaintainer;
   }
 
-  public static GivrUser getUserByIdFromDatastore(String userId) {
+  public String getUserEmail() {
+    return this.email;
+  }
+
+  public boolean isLoggedIn() {
+    return this.isLoggedIn;
+  }
+
+  public static PreparedQuery getPreparedQueryResultUserWithProperty(String propertyName, String propertyValue) {
```

Comment:
Can you leave a comment about how this is meant to be used?  I'd imagine you want to be able to have multiple query filters in a prepared query, ie;

PrepartedQuery = GiverUser.PreparedQueryBuilder().With("userId", "fjosfjs").With("isMaintainer", true).Build();

?

Reply:
Comment added!